examples/service-fastapi/app/client.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+import httpx
+
+
+### For consideration: turn this into a class with
+### feature parity for HubAuth, or even subclass HubAuth?
```

Comment:
I think both are useful - using HubAuth as in https://gist.github.com/bollwyvl/1609d764317a439dfd95f6427fbd38f8, or the no-imports implementation. We could have both.

It would be better, though, to use HubOAuth with a callback handler than HubAuth, which doesn't do the oauth flow, it only works for token-authenticated API requests or old shared-cookie auth.

Reply:
comment and README updated to say HubOAuth instead of HubAuth.  If you're okay with it, I'll leave the minimal implementation as is, and not try to create a HubOAuth with an httpx (async requests) plug-in.